File: languages/Nim/fib.nim
import times

proc fib(n: int): int =
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

let startTime = cpuTime()
discard fib(45)
let endTime = cpuTime()
echo (endTime - startTime), " seconds to execute"

File: languages/JS/fib.js
function fib(n) {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}

const startTime = Date.now();
fib(45);
const endTime = Date.now();
console.log((endTime - startTime) / 1000, " seconds to execute");
